src/Spryker/Install/Stage/Section/Command/Command.php

```diff
@@ -295,4 +300,32 @@ public function breakOnFailure(): bool
     {
         return $this->breakOnFailure;
     }
+
+    /**
+     * @return bool
+     */
+    public function hasTimeout(): bool
+    {
+        return ($this->timeout !== null);
```

Comment:
How about using 0 as default and then check on that?
Then we dont need it nullable, and it is in sync with what exec and PHP mean with 0 anyway (no timeout).